### ISSUE TITLE
REC track diagram does not reflect normative text

### ISSUE THREAD

**Issue by gsnedders** [w3c/w3process#638]
The diagram in https://www.w3.org/Consortium/Process/Drafts/#rec-track doesn't seem to be fully explainable by normative text.

Neither the [W3C Recommendation Track](https://www.w3.org/Consortium/Process/Drafts/#w3c-recommendation-track) definition, nor the [Candidate Recommendation](https://www.w3.org/Consortium/Process/Drafts/#RecsCR) definition, nor the [Transitioning to Proposed Recommendation](https://www.w3.org/Consortium/Process/Drafts/#transition-pr) section seem to actually clarify that the final CR prior to advancement has to be a CR Snapshot, which I believe (see #636) is the intention of the diagram.

**Comment by plehegar** [comment]
In [Transitioning to Candidate Recommendation](https://www.w3.org/2021/Process-20211102/#transition-cr), we have:
[[
The first Candidate Recommendation publication after approval of a [Transition Request](https://www.w3.org/2021/Process-20211102/#trans-req) is always a [Candidate Recommendation Snapshot](https://www.w3.org/2021/Process-20211102/#candidate-recommendation-snapshot).
]]

So, the first CR published is also a CR Snapshot even if it doesn't say so explicitly.

Then, in [Transitioning to Proposed Recommendation](https://www.w3.org/2021/Process-20211102/#transition-pr), we have:
[[
must not have made any [substantive changes](https://www.w3.org/2021/Process-20211102/#substantive-change) to the document since the most recent [Candidate Recommendation Snapshot](https://www.w3.org/2021/Process-20211102/#candidate-recommendation-snapshot), other than dropping features identified [at risk](https://www.w3.org/2021/Process-20211102/#at-risk).
]]

So, your last CR prior to advancement may be a CR Draft, as long as you're not trying to introduce substantive changes since the last CR Snapshot.

**Comment by plehegar** [comment]
The diagram could be improved indeed. The risk being to make it too complicated to read.

**Comment by fantasai** [comment]
Yeah, we'd have to introduce a bubble in the middle of the arrow to PR that says “CRD with only editorial changes since previous CRS” ... which seems maybe a bit too complicated to put in the diagram? There's no harm in publishing a new CRS in between, if someone interprets it that way anyway; that can be done (since it's only editorial changes) very easily.

**Comment by fantasai** [comment]
(To be honest, I'd almost prefer to require the extra CRS than complicate the diagram further, since editorial CRS updates are automatic).

**Comment by frivoal** [comment]
This is true. I am not particularly good at diagrams, so if you (or someone else) could supply an improved diagram, I would very much appreciate.

**Comment by LJWatson** [comment]
Given that the process is described in the text, I suggest making the diagram illustrative rather than detailed/descriptive. I added the ARIA to the present diagram AIRC and it adds a layer of complexity because it requires the SVG to be arranged very carefully. 

If the diagram just illustrates what's in the text without adding any additional information, then the accessibility could be simplified by either putting `role="img"` and `aria-label="Some short text description here"` on the `svg` element, or by hiding the image from screen readers entirely by putting `aria-hidden="true"` on the `svg` element instead.

**Comment by fantasai** [comment]
@frivoal I don't think we should fix the diagram to show the "CRS -> editorial changes -> CRD -> PR" path. It's really complicated to show on a diagram, and that makes the diagram less valuable because it becomes harder to parse for every other thing in it (which are all things that matter a lot more).

If someone uses the editorial CRD->PR path because it is possible (per normative text) great. And if someone only looks at the diagram and concludes they need to publish another CRS first before requesting PR, that's also totally fine. Publishing an editorial CRS is a matter of 5 minutes, it's not a big deal.

**Comment by gsnedders** [comment]
For reference, this led to both myself and @tantek being somewhat confused as to what the process allowed, due to the disagreement between the diagram and the text (and wondering whether or not we were reading the normative text correctly, not least due to it not always being clear if the distinction between CR and CRS is deliberate in places).

Given neither of us are really new to Process, to me it does feel like there is a real need for the diagram to be accurate as if we're confused I can only imagine many other people who _aren't_ used to dealing with it frequently are more so.

If publishing an editorial CRD->CRS is very simple, then I'm somewhat tempted to suggest we just eliminate the option of going from CRD to PR? (Though that's clearly not an editorial change to Process!)

**Comment by frivoal** [comment]
Separately from considering potential normative changes to the Process, I think we should mark the diagrams explicitly as informative, noting that they are a summary of the normative text, and that details may be missing.
